### PR TITLE
update README

### PR DESCRIPTION
**Pull request by nimakaviani**
[#162409859](https://www.pivotaltracker.com/story/show/162409859)

### CODE REVIEW

File: scf/README.md [hunk 1]
```diff
@@ -1,48 +1,51 @@
...
 
 - Make sure your Kubernetes cluster meets all [SCF related Kubernetes Requirements](https://github.com/SUSE/scf/wiki/How-to-Install-SCF#requirements-for-kubernetes).
 - Install [helm](https://helm.sh/)
 
 ## Deploy
 
 1. Choose a [non NFS based `StorageClass`](https://github.com/SUSE/scf/wiki/How-to-Install-SCF#choosing-a-storage-class) because MySQL does not work well with it. For additional information you can take a look at [Cloud Specifics](#cloud-specifics)
-1. Download and unzip the latest [release](https://github.com/cloudfoundry-incubator/eirini-release/releases)
-1. Create a `scf-config.yaml` file using the following template:
+1. Clone or download and unzip the latest  [Eirini Release](https://github.com/cloudfoundry-incubator/eirini-release/releases)
```
[the remark this answers is not in the document]
Comment (nimakaviani) [reply scf/README.md]: sure or we can mention that the image tag can be added to `scf-config-values.yaml`.

File: scf/README.md [hunk 2]
```diff
@@ -51,40 +54,61 @@ You are basically two `helm install's` away from deploying `SCF` with `Eirini`.
...
       # when using Bits-Service as image registry:
       BITS_SERVICE_SECRET: changeme
-      BITS_SERVICE_SIGNING_USER_PASSWORD: ehangeme
+      BITS_SERVICE_SIGNING_USER_PASSWORD: changeme
       BLOBSTORE_PASSWORD: changeme
     ```
    More information on the SCF deployment configurations can be found in the [SCF configurations docs](https://github.com/SUSE/scf/wiki/How-to-Install-SCF#configuring-the-deployment)
 
-1. To deploy use the following commands as explained in the [SCF documentation](https://github.com/SUSE/scf/wiki/How-to-Install-SCF#deploy-using-helm). In the commads `uaa` and `cf` are the helm charts you unzipped from the `eirini-scf-release`:
+1. To deploy `cd` into `eirini-release/scf/helm` and use the following commands as explained in the [SCF documentation](https://github.com/SUSE/scf/wiki/How-to-Install-SCF#deploy-using-helm). In the commands `uaa` and `cf` refer to the helm chart directories from `eirini-release/scf/helm`:
```
[the remark this answers is not in the document]
Comment (nimakaviani) [reply scf/README.md]: 👍 updated it accordingly.

File: scf/README.md [hunk 3]
```diff
@@ -51,40 +54,61 @@ You are basically two `helm install's` away from deploying `SCF` with `Eirini`.
...
     ```bash
-    $ helm install uaa --namespace uaa --values scf-config.yaml --name uaa
+    $ helm install uaa --namespace uaa --values scf-config-values.yaml --name uaa
     $ SECRET=$(kubectl get pods --namespace uaa -o jsonpath='{.items[?(.metadata.name=="uaa-0")].spec.containers[?(.name=="uaa")].env[?(.name=="INTERNAL_CA_CERT")].valueFrom.secretKeyRef.name}')
     $ CA_CERT="$(kubectl get secret $SECRET --namespace uaa -o jsonpath="{.data['internal-ca-cert']}" | base64 --decode -)"
     $ helm install cf --namespace scf --name scf --values scf-config-values.yaml --set "secrets.UAA_CA_CERT=${CA_CERT}"
     ```
+
+   **NOTE**: When getting the uaa `SECRET`,
+be aware that secrets are versioned and the numerical suffix on the secret name will change if you upgrade the chart.
```
[the remark this answers is not in the document]
Comment (nimakaviani) [reply scf/README.md]: removed

File: scf/README.md [hunk 4]
```diff
@@ -1,48 +1,56 @@
...
+      # you can ignore this porperty if use_registry_ingress is set to `false`
+      ingress_endpoint: <ingress-endpoint>
+
+      # the namespace eirini/opi schedules the apps to.
+      namespace: <kubernetes-namespace>
+
+      # if you clone the eirini-release repo you need to explicitly specify the image tag.
+      # when downloading the release tarball, `image_tag` is already set to `latest`, in
+      # which case you can drop the following property.
+      image_tag: latest
```
Comment (suhlig) [inline scf/README.md]: We actually started setting this to a particular value, but we'll fix this after merging the PR.

**Pull request merged**